src/nvidia-cuda/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+install_cuda=${INSTALL_CUDA:-"true"}
+install_cudnn=${INSTALL_CUDNN:-"true"}
+
+if [ "$(id -u)" -ne 0 ]; then
+    echo -e 'Script must be run as root. Use sudo, su, or add "USER root" to your Dockerfile before running this script.'
+    exit 1
+fi
+
+# Download NVIDIA's keyring package
+ubuntu_version="$(lsb_release -sr | sed 's/\.//g')"
+keyring_repo="https://developer.download.nvidia.com/compute/cuda/repos/ubuntu$ubuntu_version/x86_64"
+keyring_package="cuda-keyring_1.0-1_all.deb"
+keyring_package_url="$keyring_repo/$keyring_package"
+keyring_package_path="$(mktemp -d)"
+keyring_package_file="$keyring_package_path/$keyring_package"
+wget -O "$keyring_package_file" "$keyring_package_url"
+
+# Install NVIDIA's keyring package
+apt-get install -yq "$keyring_package_file"
+apt-get update -yq
+
+if [ "$install_cuda" = "true" ]; then
+    echo "Installing CUDA libraries..."
+    apt-get install -yq cuda-libraries-11-7
```

Comment:
Do we need to expose the version of these libraries as an option for users to pick?

Reply:
Is this alone enough? 

e.g., this is how Nvidia's base image installs: https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/11.7.0/ubuntu2004/runtime/Dockerfile#L34

It also installs `libcublas-11-7`, `cuda-nvtx-11-7`, and `libcusparse-11-7`.  I ask since I've used the Dockerfile logic above myself so I know that works.

Reply:
`cuda-libraries-11-7` depends on a handful of library packages including `libcublas-11-7` and `libcusparse-11-7`, and the test I added specifically checks for these libraries: https://github.com/devcontainers/features/blob/62bcebaeabae49ebab62672747d4f1677399b4f3/test/nvidia-cuda/test.sh#L9-L15

<img width="718" alt="Screen Shot 2022-08-15 at 3 34 27 PM" src="https://user-images.githubusercontent.com/19893438/184732552-343943c7-1bdc-4427-8150-a37019eebbb3.png">

Reply:
We could add an option for NVTX, but I don't think it's needed by default: https://docs.nvidia.com/nsight-visual-studio-edition/nvtx/index.html

Reply:
I added an option for NVTX